poet/__main__.py

```diff
@@ -1,6 +1,5 @@
-import sys
-
 from .poet import *
```

Comment:
Because this module don't need of the `sys` module. So, if someone import the package `poet` it will import `sys` too without necessity.